ibm_catalog.json

```diff
@@ -0,0 +1,231 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-watsonx-ai",
+        "label": "watsonx.ai",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "ai",
+          "solution"
+        ],
+        "keywords": [
+          "watsonx ai",
+          "watsonx.ai",
+          "watsonx.ai project",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "ai"
+        ],
+        "short_description": "Creates and configures IBM watsonx.ai Project",
+        "long_description": "This architecture supports creating and configuring the instances of watsonx.ai Studio, watsonx.ai Runtime and creates a KMS encrypted watsonx.ai Project.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-ai/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-ai/main/images/watsonx-ai-da.svg",
```

Comment:
The images directory is missing which should contain `watsonx.ai` icon and is referred in the `"offering_icon_url"` section.

Reply:
oops, missed that. thanks. Added now.